src/main/asciidoc/_chapters/upgrading.adoc

```diff
@@ -601,10 +601,10 @@ The HBase shell command relies on a bundled JRuby instance. This bundled JRuby b
 
 The HBase shell command now ignores the '--return-values' flag that was present in early HBase 1.4 releases. Instead the shell always behaves as though that flag were passed. If you wish to avoid having expression results printed in the console you should alter your IRB configuration as noted in the section <<irbrc>>.
 
-[[upgrade2.0.coprocessors]]
+[[upgrade2.0.coprocessors-changes]]
 .Coprocessor APIs have changed in HBase 2.0+
 
-All Coprocessor APIs have been refactored to improve supportability around binary API compatibility for future versions of HBase. If you or applications you rely on have custom HBase coprocessors, you should read link:https://issues.apache.org/jira/browse/HBASE-18169[the release notes for HBASE-18169] for details of changes you will need to make prior to upgrading to HBase 2.0+.
```

Comment:
The old wording is fine, I suggest keeping that.

Reply:
To be completely honest the original wording sounded incorrect to me but now after I re-read it again, it makes sense. I'll revert this change.